app/src/main/java/com/gh4a/activities/IssueListActivity.java

```diff
@@ -114,15 +114,20 @@ public static Intent makeIntent(Context context, String repoOwner, String repoNa
     private static final String LIST_QUERY = "is:%s %s repo:%s/%s %s %s %s %s";
     private static final String SEARCH_QUERY = "is:%s %s repo:%s/%s %s";
 
-    private static final int[] TITLES = new int[] {
+    private static final int[] ISSUE_TITLES = new int[] {
         R.string.open, R.string.closed
     };
 
+    private static final int[][] ISSUE_HEADER_COLOR_ATTRS = new int[][] {
+        { R.attr.colorIssueOpen, R.attr.colorIssueOpenDark },
+        { R.attr.colorIssueClosedCompleted, R.attr.colorIssueClosedCompletedDark }
+    };
```

Comment:
Since the issue activity distinguishes between 'open', 'closed' and 'completed' now, I think the issue list activity should do the same.

Reply:
I don't agree on this... "completed" is the reason why an issue has been closed, it isn't a third state.
Furthermore, if we follow this logic, the three tabs should be called "Open",  "Not planned" and "Completed". But even in that way it would look overkill IMHO, considering how the GitHub UI displays the new close reasons.

Reply:
'Completed' basically is the equivalent of the 'merged' PR state: 'closed as done' ... treating it that way in the UI doesn't seem like overkill to me, but rather more consistent?

Reply:
> 'Completed' basically is the equivalent of the 'merged' PR state: 'closed as done'

Mmhh... I believe it's a bit of a stretch to see "closed as completed" in the same way as the Merged state.
The GitHub UI emphasizes it differently (there's no "Completed" purple pill for issues like the Merged pill for PRs) and there's also the fact that all old issues are displayed as "closed as completed" while they might have been closed for other reasons... this is why I don't like having a "hard" separation in the UI.
My idea in this PR was to replicate the GH web UI behavior in the app, with the only difference that _not planned_ is red instead of grey.

> show all closed issues, with red header

I guess it could be fine, but what about using purple for all issues instead? Red is not used anymore for issues in GitHub UI and _completed_ has become the "default" closed reason.